app/views/spree/admin/premade_carts/_add_to_premade_cart_link.html.erb

```diff
@@ -0,0 +1,8 @@
+<% premade_cart = Spree::PremadeCart.first %>
```

Comment:
This seems like an unusual assumption; is the expected case that the customer will only have one premade cart?

Reply:
Sounds good then.